src/Components/TrainingPage.js

```diff
@@ -1,21 +1,43 @@
-import React from 'react';
-import { StyleSheet, View, Text } from 'react-native';
+import React, { Component } from 'react';
+import { StyleSheet, ScrollView, Text } from 'react-native';
 import { connect } from 'react-redux';
+import DrillList from './DrillList';
 import theme from '../styles/theme.style';
 
+const getGoals = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  return Math.max(...nbPlayersList);
+};
+
+const getEquipment = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
```

Comment:
Same with equipment?

Reply:
I'm not sure how we want to use and store equipment for now..so I leave it for now. I would like to discuss this so I can add it in another PR